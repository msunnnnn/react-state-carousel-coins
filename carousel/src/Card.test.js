import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon";

const TEST_PHOTO = TEST_IMAGES[1];

it("renders without crashing", function () {
  render(
  <Card
    caption={TEST_PHOTO.caption}
    src={TEST_PHOTO.src}
    currNum="1"
    totalNum="1" />
  );
});


it("matches snapshot", function () {
  const { container } = render(
    <Card
      caption={TEST_PHOTO.caption}
      src={TEST_PHOTO.src}
      currNum="1"
      totalNum="1" />);
  expect(container).toMatchSnapshot();

});