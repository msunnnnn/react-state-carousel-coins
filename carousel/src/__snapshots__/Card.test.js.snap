// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testing image 2
    </h4>
    <img
      alt="testing image 2"
      class="Card-image"
      src="test2.com"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      1
      .
    </small>
  </div>
</div>
`;
